README.md

```diff
@@ -500,20 +499,25 @@ websocket.unsubscribe({ channels: ['full'] });
 websocket.subscribe({ product_ids: ['LTC-USD'], channels: ['ticker', 'user'] });
 
 websocket.subscribe({
-  channels: [{
-    name: 'user',
-    product_ids: ['ETH-USD']
-  }]
+  channels: [
```

Comment:
I ran prettier on the entire codebase a few days ago, a bit surprised that there were still errors.